msbuild/domoticz.vcxproj

```diff
@@ -618,6 +618,8 @@
     <ClInclude Include="..\main\WindCalculation.h" />
     <ClInclude Include="..\smtpclient\SMTPClient.h" />
     <ClInclude Include="..\sqlite\sqlite3.h" />
+    <ClInclude Include="..\tinyxpath\tinyxml.h" />
```

Comment:
I think you also need to add the tinyxpath folder to the additional includes folder for both debug/release